src/coreclr/pal/src/thread/process.cpp

```diff
@@ -2212,6 +2210,9 @@ PROCCreateCrashDump(
     INT cbErrorMessageBuffer,
     bool serialize)
 {
+#if defined(TARGET_IOS)
```

Comment:
we should disable this on tvos too

Reply:
I didn't enable tvOS in this PR. It requires more work and testing due to lack of Mach signal API.

(Previous versions of this PR enabled the tvOS compilation at one point. I may resubmit them separately at some point for a more thorough review. Mixing them with the rest of the iOS bring up made it unreviewable.)

Reply:
yes I just wanted to call out so we don't forget to update this spot